frontend/src/concepts/pipelines/content/createRun/RunForm.tsx

```diff
@@ -72,6 +72,7 @@ const RunForm: React.FC<RunFormProps> = ({ data, runType, onValueChange }) => {
           descriptionFieldId="run-description"
           data={data.nameDesc}
           setData={(nameDesc) => onValueChange('nameDesc', nameDesc)}
+          maxLength={MAX_CHARACTER_LIMIT}
```

Comment:
On second thoughts, can we not have this directly in `NameDescription` field? Instead of trickling this down from here?

Reply:
`NameDescriptionField` is used in other components as well. Using MAX_CHARACTER_LIMIT directly will affect the max char limit of other components.